bbc/simorgh
# WSTEAM1-1034 Implement UX/UI for Uzbek scriptLink

Resolves JIRA [WSTEAM1-1034](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1034)

Overall changes
======
 Adds logic to only render the `scriptLink` button if the article is published through optimo and translations exist for it.
 
Code changes
======

- _A bullet point list of key code changes that have been made._

Testing
======
1. Visit http://localhost:7080/uzbek/articles/cxj3rjxm6r0o/lat and observe the scriptLink button, test it works. Comment out the translations for the script link [here](https://github.com/bbc/simorgh/blob/WSTEAM1-1034-script-link-ui-uzbek/src/app/lib/config/services/uzbek.ts#L362C3-L382C5) and observe the button no longer being rendered
2. Visit http://localhost:7080/uzbek/uzbekistan-53263099 and observe there being no button rendered


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
